.github/workflows/release.packages.yml

```diff
@@ -0,0 +1,472 @@
+name: release.packages.yml
+run-name: Release NPM packages
+on:
+  pull_request:
+    types:
+      - closed
+    branches:
+      - main
+
+
+jobs:
+
+  ### This job is used to get all changed files in the PR
+
+  changed-files-job:
+    name: Get files changed in PR
+    runs-on: ubuntu-latest
+    outputs:
+      all_changed_files: ${{ steps.changed-files.outputs.all_changed_files }}
+    steps:
+      - name: Get changed files
+        id: changed-files
+        uses: tj-actions/changed-files@ed68ef82c095e0d48ec87eccea555d944a631a4c # v46
+        with:
+          # Limit the scope of the changed files to only those in the packages directory
+          files: packages/**
+      - name: List all changed files
+        env:
+          ALL_CHANGED_FILES: ${{ steps.changed-files.outputs.all_changed_files }}
+        run: |
+          for file in ${ALL_CHANGED_FILES}; do
+            echo "$file was changed"
+          done
+
+
+  ### Dev packages
+
+  biome-config:
+    name: Release biome-config
+    if: |
+      always() &&
+      !contains(needs.*.result, 'failure') &&
+      !contains(needs.*.result, 'cancelled') &&
```

Comment:
These conditions are always required to be able to check for skipped jobs.

IE: if your new PR has changes only in 1 package, which has dependency on `biome-config`, a release of `biome-config` could be skipped or should be released first.